public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -123,18 +139,18 @@ export const AnomalyHistory = (props: AnomalyHistoryProps) => {
         );
       } finally {
         setIsLoadingAnomalyResults(false);
-        fetchRawAnomalyResults(false);
       }
     }
 
+    fetchRawAnomalyResults(isHCDetector);
```

Comment:
according to `fetchRawAnomalyResults()`, the param is related to loading or not - why does `isHCDetector` impact this?

Reply:
For HC detector, we are relying on raw data to get data for heatmap, that is why we want to show the loading spinner; For single entity detector, fetching raw data should not impact loading state. If fetching large time range data, we use `isLoadingAnomalyResults` to control loading spinner; if fetching small time range data, it is every quick for single entity detector, I think skipping loading spinner is fine for small time range case. I can change to even show loading spinner for small time range in case of single entity detector, it just added a few more `if - else`. Please feel free to let me know your thoughts.

Reply:
I see, makes sense, thanks for explaining. I think it is good as is - I haven't noticed loading issues while testing.